codegen2.py

```diff
@@ -146,6 +146,7 @@ def generated_h(classes, libname):
 #include <wx/clrpicker.h>
 #include <wx/datectrl.h>
 #include <wx/filepicker.h>
+#include <wx/filedlg.h>
```

Comment:
Please insert this alphabetical order if you don't strong preference to this order.